src/mofaflex/_core/mofaflex.py

```diff
@@ -133,6 +136,15 @@ class ModelOptions(_Options):
     nonnegative_weights: dict[str, bool] | bool = False
     """Non-negativity constraints for weights for each view (if dict) or for all views (if bool)."""
 
+    guiding_vars_weight_priors: dict[str, str] | str | None = None
```

Comment:
I think this should default to `Normal`. It will be ignored anyway if `DataOptions.guiding_vars_obs_keys is None`. Also, instead of `str` for the type hint, use `Literal["Normal", "otherdistribution"]` to have proper docs for the allowed values. Look at `weight_prior` and `factor_prior`.